src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -457,8 +457,8 @@ class WorkItemListPage {
     return this.workItemKebabMoveToBottomButton (parentElement).click();
   }
   /**Iterations Page object model */
-  iterationAddButton  (){
-    return element(by.css('.pficon-add-circle-o.fa-icon'));
+  iterationAddButton (){
+    return element(by.id('add-iteration-icon'));
   }
   clickIterationCreateLabel  (){
     return element(by.id("add-iteration")).click();
```

Comment:
None of the tests will run on my environment (either pass or fail). On npm install, the dependencies were overwritten and broke my environment. Joshua tried to fix, but could not (he's investigating a fix). Would you be able to commit the proper changes for me? It'd be greatly appreciated.